src/github.com/getlantern/lantern-ui/app/js/services.js

```diff
@@ -56,97 +56,40 @@ angular.module('app.services', [])
     }
 
     var fnList = {
-      'GeoLookup': function(data) {
-        console.log('Got GeoLookup information: ', data);
-        if (data && data.Location) {
-            model.location = {};
-            model.location.lon = data.Location.Longitude;
-            model.location.lat = data.Location.Latitude;
-            model.location.resolved = true;
+      'settings': function(settings) {
+        console.log('Got Lantern default settings: ', settings);
+        if (settings && settings.version) {
+            // configure settings
+            // set default client to get-mode
+            model.settings = {};
+            model.settings.mode = 'get';
+            model.settings.version = settings.version + " (" + settings.revision_date + ")";
         }
-      },
-      'Settings': function(data) {
-        if (data.Settings) {
-          var settings = data.Settings;
-          console.log('Got Lantern default settings: ', settings);
-          if (settings && settings.Version) {
-              // configure settings
-              // set default client to get-mode
-              model.settings = {};
-              model.settings.mode = 'get';
-              model.settings.version = settings.Version + " (" + settings.RevisionDate + ")";
-          }
-
-          if (settings.AutoReport) {
-              model.settings.autoReport = true;
-                  $rootScope.trackPageView();
-          }
-
-          if (settings.AutoLaunch) {
-              model.settings.autoLaunch = true;
-          }
-
-          if (settings.ProxyAll) {
-              model.settings.proxyAll = true;
-          }
 
-          if (settings.SystemProxy) {
-              model.settings.systemProxy = true;
-          }
+        if (settings.auto_report) {
+          model.settings.auto_report = true;
+          $rootScope.trackPageView();
```

Comment:
Yeah, so there's no real reason to keep it here right? then we can move it to a better place. This part of code is only fired when the backend sends a message, and that should happen every time settings are changed, that's why it's pretty odd to have it here.